src/view/Office.vue

```diff
@@ -313,47 +327,35 @@ export default {
 			display: flex;
 			padding: 4px;
 
-			.avatardiv {
+			::v-deep .avatardiv {
 				margin-left: -15px;
 				box-shadow: 0 0 3px var(--color-box-shadow);
 			}
-
 		}
 
-		.icon-menu-sidebar {
+		&__icon-menu-sidebar {
 			background-image: var(--icon-menu-sidebar-000) !important;
 		}
 	}
 
-	#richdocuments-wrapper {
-		width: 100%;
-		height: 100%;
-		top: 0;
-		left: 0;
-		position: absolute;
-		z-index: 100000;
-		max-width: 100%;
-		display: flex;
-		flex-direction: column;
-		background-color: var(--color-main-background);
-		transition: opacity .25s;
-
-		.viewer & {
-			height: 100vh;
-			top: -50px;
-		}
+	::v-deep .viewer & {
+		height: 100vh;
+		top: -50px;
```

Comment:
At least this is not effective. The height and top are not applied to `.office-viewer`.

![collab](https://user-images.githubusercontent.com/11291457/189377092-0660380b-5681-4de2-be92-fbd0bc178957.png)


I suspect `::v-deep` is useless when you select something outside the scope. Here `.viewer` is outside `&` aka `.office-viewer`.

Reply:
You are right. Deep means applying to children. Not sibling or parent elements.